src/write.c

```diff
@@ -993,7 +996,8 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
     for (uint32_t itemIndex = 0; itemIndex < encoder->data->items.count; ++itemIndex) {
         avifEncoderItem * item = &encoder->data->items.item[itemIndex];
 
-        avifBoxMarker infe = avifRWStreamWriteFullBox(&s, "infe", AVIF_BOX_SIZE_TBD, 2, 0);
+        uint32_t flags = item->hiddenImage ? 1 : 0;
+        avifBoxMarker infe = avifRWStreamWriteFullBox(&s, "infe", AVIF_BOX_SIZE_TBD, 2, flags);
```

Comment:
Joe: There is an `avifRWStreamWriteFullBox(s, "infe", AVIF_BOX_SIZE_TBD, 2, 0)` call in `avifEncoderWriteTrackMetaBox()`.  Should I also update that call to consider `item->hiddenImage`?

Reply:
Anything that is written for a `trak`'s `meta` box is going to be considered a part of the presentation as it will be a frame in the sequence, so I think not. If anything, it should always be false when writing out a track's meta, right?